.gitignore

```diff
@@ -8,3 +8,4 @@ cmd/**/debug
 hack/**/debug
 debug.test
 *.iml
+examples/k8s-jobs1.yaml
```

Comment:
Why is this under .gitignore?

Reply:
Sorry these files by mistake got it in  PR I will revert it